### ISSUE TITLE
Github connection pending handshake

### ISSUE THREAD

**Issue by vinycoolguy2015** [AppGambitStudio/terraform-apprunner#1]
Github connection stuck in pending handshake and service creation failed with "Failed to retrieve the latest commit from your source repository. Be sure you configure your service with a valid connection to your GitHub repository" message

**Comment by Prashant-jumpbyte** [comment]
Yes, because for now terraform didn't support github connection directly Please check https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/apprunner_connection

For now, You have to configure your Github connection manually.

**Comment by vinycoolguy2015** [comment]
As per the doc "After creation, you must complete the authentication handshake using the App Runner console." but complete handshake button is greyed out and it starts getting deleted as soon as GitHub connection is created
